streaming_data_types/area_detector_NDAr.py

```diff
@@ -1,11 +1,51 @@
 from typing import Union
-from streaming_data_types.fbschemas.NDAr_NDArray_schema import NDArray, NDAttribute
+import flatbuffers
+from streaming_data_types.fbschemas.NDAr_NDArray_schema import NDArray
 from streaming_data_types.utils import check_schema_identifier
 from collections import namedtuple
+import time
 
 FILE_IDENTIFIER = b"NDAr"
 
 
+def serialise_ndar(
+    id: str,
+    dims: list,
+    data_type: int,
+    data: list,
+) -> bytes:
+    builder = flatbuffers.Builder(1024)
+
```

Comment:
```suggestion
builder.ForceDefaults(True)

```
Add this line to work around a bug in the flat buffers library.